compiler/damlc/daml-doc/src/DA/Daml/Doc/Driver.hs

```diff
@@ -72,11 +72,12 @@ damlDocDriver cInputFormat ideOpts output cFormat prefixFile options files = do
 
     case cFormat of
             Json -> write output $ T.decodeUtf8 . BS.toStrict $ AP.encodePretty' jsonConf docData
-            Rst  -> write output . renderFinish . mconcat $ map renderSimpleRst docData
+            Rst  -> write output . renderPage . mconcat $ map renderSimpleRst docData
             Hoogle   -> write output . T.concat $ map renderSimpleHoogle docData
-            Markdown -> write output . renderFinish . mconcat $ map renderSimpleMD docData
-            Html -> sequence_
-                [ write (output </> hyphenated (unModulename md_name) <> ".html") $ renderSimpleHtml m
-                | m@ModuleDoc{..} <- docData ]
-                    where hyphenated = T.unpack . T.replace "." "-"
-    putStrLn "Done"
+            Markdown -> write output . renderPage . mconcat $ map renderSimpleMD docData
+            Html -> do
+                let ro_mode = RenderToFolder output
+                    ro_format = Html
+                    ro_title = Nothing
+                    ro_template = Nothing
+                renderDocs RenderOptions{..} docData
```

Comment:
I'm not the biggest fan of using record wild cards to construct value when the proper record syntax would be good enough. I prefer something more explicit like
```haskell
let opts = RenderOptions
      { ro_mode = RenderToFolder output
      ; ...
      }
```

Reply:
:+1: that looks good!